ovirt_imageio/tool/download.py

```diff
@@ -0,0 +1,77 @@
+# ovirt-imageio
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+"""
+Download commands.
+"""
+
+from contextlib import closing
+
+from .. import client
+from .. _internal.units import KiB, MiB
+
+from . import ovirt
+from . import options
+
+
+def register(parser):
+    cmd = parser.add_sub_command(
+        "download-disk",
+        help="Download disk",
+        func=download_disk)
+
+    cmd.add_argument(
+        "-f", "--format",
+        choices=("raw", "qcow2"),
+        default="qcow2",
+        help="Download image format (default qcow2).")
+
+    cmd.add_argument(
+        "--max-workers",
+        type=options.Size(min_value=1, max_value=8),
+        default=client.MAX_WORKERS,
+        help=f"Maximum number of workers (default {client.MAX_WORKERS}).")
+
+    cmd.add_argument(
+        "--buffer-size",
+        type=options.Size(min_value=64 * KiB, max_value=16 * MiB),
```

Comment:
It would be nice if the help text would include these limits, otherwise the user has to guess and retry until he's inside the legal interval.
Same for --max-workers, but the buffer-size is probably more variable.

Reply:
This is unlikely to be a real issue since users are not expected to play with this, but it should be easy to expose the limits using the validator limits.

Reply:
Done in current version.